doc/source/running.rst

```diff
@@ -84,6 +84,13 @@ For |PBM| version 1.3.4 and earlier, run the |pbm-list| command and you will see
 Restoring a backup
 --------------------------------------------------------------------------------
 
+.. warning::
+
+   Backups made with |PBM| v1.4.1 and earlier are incompatible for restore with |PBM| v1.5.0 and later. This is because processing of system collections ``Users`` and ``Roles`` has changed: in v1.5.0, ``Users`` and ``Roles`` are copied to temporary collection during backup and must be present in the backup during restore. In v1.4.1 and earlier, ``Users`` and ``Roles`` are copied to a temporary collection during restore. Therefore, restoring from these backups with |PBM| v1.5.0 isn't possible. 
```

Comment:
Maybe it makes sense not to mention any specific version prior to v.1.5.0. I mean instead phrasing: 
```
Backups made with |PBM| prior to v1.5.0 are incompatible for restore with |PBM| v1.5.0 and later.
```
And so on.

Just in case we find some critical bug or gonna have some other reason to release the patch for v1.4 - v1.4.2. In that case, have to remember to change docs here etc.

Plus it more obvious - everything prior 1.5.0 isn't compatible.

Reply:
@akira-kurogane what do you think? ^^^

Reply:
I didn't see a problem with saying 1.4.1 explicitly, given this is a patch for the imminent release of v1.5.0 and we don't have separate version branches, the chances of there being a 1.4.2 are now virtually zero.

Having said that, I do like the suggested edit more. It's shorter to read, simple.